packages/help-center/src/components/test/back-button.tsx

```diff
@@ -16,40 +17,39 @@ jest.mock( 'react-router-dom', () => ( {
 	} ),
 } ) );
 
-function renderWrapper( props: Props ) {
-	return shallow( <BackButton { ...props } /> );
-}
-
 describe( 'BackButton', () => {
-	describe( 'when the user clicks the back button', () => {
-		describe( 'and the system decides to navigate back home', () => {
-			it( 'navigates back home', () => {
-				const wrapper = renderWrapper( { backToRoot: true } );
-				const button = wrapper.find( Button );
-				button.simulate( 'click' );
-				expect( mockHistoryPush.mock.calls.length ).toBe( 1 );
-			} );
-		} );
-
-		describe( 'and the system decides to navigate to the previous page', () => {
-			it( 'navigates to the previous page', () => {
-				const wrapper = renderWrapper( { backToRoot: false } );
-				const button = wrapper.find( Button );
-				button.simulate( 'click' );
-				expect( mockHistoryGoBack.mock.calls.length ).toBe( 1 );
-			} );
-		} );
-
-		describe( 'and the system decides to navigate to a custom page', () => {
-			it( 'navigates to the custom page', () => {
-				const onClickSpy = jest.fn();
-				const wrapper = renderWrapper( { onClick: onClickSpy } );
-
-				const button = wrapper.find( Button );
-				button.simulate( 'click' );
-
-				expect( onClickSpy.mock.calls.length ).toBe( 1 );
-			} );
-		} );
+	afterEach( () => {
+		mockHistoryGoBack.mockClear();
+		mockHistoryPush.mockClear();
+	} );
+
+	it( 'navigates to the root when back to root is true', async () => {
+		render( <BackButton backToRoot /> );
+
+		const btn = screen.getByRole( 'button' );
+		await userEvent.click( btn );
```

Comment:
It's generally recommended to run `user.setup()` first.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/70917/commits/f20b73dc1ab23dce4ae5b187ed1a335424c65813